### PR TITLE
PCHR-2594: Improve Modal Styling

### PR DESCRIPTION
**Pull request by igorpavlov-zz**
## Overview

This PR is linked with https://github.com/civicrm/civihr/pull/2152 and https://github.com/compucorp/org.civicrm.shoreditch/pull/44 and adds styles required for rendering responsive Modal properly.

## Technical Details

```css
/* Imports responsive modal from Shoreditch */
@import "SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/overrides/partials/modals-xs-screen";
```

```css
/* Adds a background colour to .modal-content */
.modal-content,
  /* ... */ {
    background: $crm-grayish-blue;
}
```

### CODE REVIEW

**Review by AkA84 — CHANGES_REQUESTED** [review]
The path of the imported file should be changed, see https://github.com/compucorp/org.civicrm.shoreditch/pull/44

**Comment by igorpavlov-zz** [comment]
I cannot neither fix conflicts nor rebase properly. Shoreditch Staging misses some commits (date-picker) and SSP la-milestone-3 requires that commit to sync css properly. Since I need to sync with Shoreditch staging (which has my changes) I need to wait for my Shoreditch being merged (https://github.com/compucorp/org.civicrm.shoreditch/pull/44) and synced, then I will be able to sync SSP

**Comment by deb1990** [comment]
Looks fine, but will approve after the sync is done.

**Review by reneolivo — APPROVED** [review]

**Pull request merged**
